xrdp/xrdp_wm.c

```diff
@@ -695,11 +695,24 @@ xrdp_wm_init(struct xrdp_wm *self)
     }
     else
     {
+#ifndef XRDP_LOGINFIRST
         g_writeln("   xrdp_wm_init: no autologin / auto run detected, draw login window");
         xrdp_login_wnd_create(self);
+#else
+        g_writeln("   xrdp_wm_init: no autologin / auto run detected, draw first stage login window");
```

Comment:
Please look at this as a user. What is "first stage login window"? How many login stages do you have? What's wrong with the existing message?